src/ar/dilar/src/eu/kanade/tachiyomi/extension/ar/dilar/Dilar.kt

```diff
@@ -23,4 +38,30 @@ class Dilar : Gmanga(
 
         return releases.map { it.toSChapter() }
     }
+
+    override fun setupPreferenceScreen(screen: PreferenceScreen) {
+        val mirrorPref = ListPreference(screen.context).apply {
+            key = MIRROR_PREF_KEY
+            title = MIRROR_PREF_TITLE
+            entries = MIRROR_PREF_ENTRY_VALUES
+            entryValues = MIRROR_PREF_ENTRY_VALUES
+            setDefaultValue(MIRROR_PREF_DEFAULT_VALUE)
+            summary = "%s"
+
+            setOnPreferenceChangeListener { _, _ ->
+                Toast.makeText(screen.context, RESTART_TACHIYOMI, Toast.LENGTH_LONG).show()
+                true
+            }
+        }
+        screen.addPreference(mirrorPref)
+    }
+
+    private fun MirrorPref(): String = preferences.getString(MIRROR_PREF_KEY, MIRROR_PREF_DEFAULT_VALUE)!!
+    override val baseUrl = MirrorPref()
```

Comment:
you removed the CI thing...

Reply:
@AwkwardPeak7 could you please explain what is the purpose of that CI thing?